src/lib/styles/stylesheet-processor.ts

```diff
@@ -21,15 +21,10 @@ export class StylesheetProcessor extends ComponentStylesheetBundler {
     // By default, browserslist defaults are too inclusive
     // https://github.com/browserslist/browserslist/blob/83764ea81ffaa39111c204b02c371afa44a4ff07/index.js#L516-L522
     // We change the default query to browsers that Angular support.
-    // https://angular.io/guide/browser-support
```

Comment:
NIT: since the browserlist is static and not auto generated, might as well, put the supported browsers queries directly in `browserslist.defaults` instead of creating a new file.

Reply:
Had a chat offline and decided to keep the separate file just to make it easier to reference and discover when bumping browser versions during releases.